src/routes/reference/mlpclassifier.svx

```diff
@@ -13,36 +13,16 @@ flair: reference
 category: Analyse Data
 ---
 
-The `MLPClassifier` is an object that can be used to perform _classification_. In machine learning, _classification_ refers to an algorithm that is trained to predict what category, or _class_, an input belongs to. 'MLP' stands for multi-layer perceptron which is a type of neural network.
+<script>
```

Comment:
I find these pages neat but a little "compact" in that information is not enumerated very clearly. Like... which order should I take these in? etc... I think that anywhere there is a video we could be using richer content and components such as <YouTube />.

Reply:
I also think a generalisable graphic might help be an evocative entry point versus quite a lot of white space and minimal text.